### PR TITLE
[BULK UPDATE] DocuTune - Rebranding links

### PR DESCRIPTION
**Pull request by alexbuckgit**
**Global effort for the Microsoft Learn platform rebrand**

The Microsoft team is applying changes to content for the Microsoft Learn platform rebrand. Changes include modifications to current Microsoft Learn and Microsoft Docs terminology as well as associated domain name and content URL path changes.

**DO NOT MERGE THIS PULL REQUEST WHILE IN DRAFT STATUS!** Some of the changes to links and other content are not valid until after go-live.

Please review this draft PR and provide any necessary comments as soon as possible. Because these fixes are tightly scoped, we have approval to automatically merge these changes to the default branch for publication.

This PR was generated via DocuTune. It includes only targeted fixes and minor formatting adjustments.

CorrelationId: d8c4c3ac-d49a-4c95-9ddc-9822ca297907
InitiativeId: docutune-rebranding-2022-09
PointOfContact: Alex Buck

### CODE REVIEW

**Comment by alexbuckgit** [comment]
#docutune-draft

File: aspnetcore/blazor/components/cascading-values-and-parameters.md [hunk 1]
```diff
@@ -742,7 +742,7 @@ Similar to a regular component parameter, components accepting a cascading param
 ```razor
 <main>
     <div class="top-row px-4">
-        <a href="https://docs.microsoft.com/aspnet/" target="_blank">About</a>
+        <a href="https://learn.microsoft.com/aspnet/" target="_blank">About</a>
```
Comment (guardrex) [inline aspnetcore/blazor/components/cascading-values-and-parameters.md:745]: Revert this one, as it's locked into the 7.0 template.

https://github.com/dotnet/aspnetcore/blob/main/src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/Shared/MainLayout.NoAuth.razor#L12

**Review by guardrex — CHANGES_REQUESTED** [review]

File: aspnetcore/fundamentals/middleware/extensibility/samples/2.x/MiddlewareExtensibilitySample/README.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
 # ASP.NET Core Middleware Extensibility Sample
 
-This sample demonstrates the scenarios described in [Factory-based middleware activation in ASP.NET Core](https://docs.microsoft.com/aspnet/core/fundamentals/middleware/middleware-extensibility).
+This sample demonstrates the scenarios described in [Factory-based middleware activation in ASP.NET Core](https://learn.microsoft.com/aspnet/core/fundamentals/middleware/middleware-extensibility).
```
Comment (annmariehitchcock) [inline aspnetcore/fundamentals/middleware/extensibility/samples/2.x/MiddlewareExtensibilitySample/README.md:3]: Adding to broken link tab, try https://learn.microsoft.com/aspnet/core/fundamentals/middleware/extensibility

File: aspnetcore/fundamentals/middleware/extensibility/samples/3.x/MiddlewareExtensibilitySample/README.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
 # ASP.NET Core Middleware Extensibility Sample
 
-This sample demonstrates the scenarios described in [Factory-based middleware activation in ASP.NET Core](https://docs.microsoft.com/aspnet/core/fundamentals/middleware/middleware-extensibility).
+This sample demonstrates the scenarios described in [Factory-based middleware activation in ASP.NET Core](https://learn.microsoft.com/aspnet/core/fundamentals/middleware/middleware-extensibility).
```
Comment (annmariehitchcock) [inline aspnetcore/fundamentals/middleware/extensibility/samples/3.x/MiddlewareExtensibilitySample/README.md:3]: Same as above, adding to broken link tab, try https://learn.microsoft.com/aspnet/core/fundamentals/middleware/extensibility

File: aspnetcore/razor-pages/razor-pages-conventions/samples/2.x/SampleApp/README.md [hunk 4]
```diff
@@ -1,12 +1,12 @@
 # ASP.NET Core Model Providers Sample
 
-This sample illustrates use of Razor Pages custom route and page model providers. This sample demonstrates the features described in the [Razor Pages route and app conventions](https://docs.microsoft.com/aspnet/core/razor-pages/razor-pages-convention-features) topic.
+This sample illustrates use of Razor Pages custom route and page model providers. This sample demonstrates the features described in the [Razor Pages route and app conventions](https://learn.microsoft.com/aspnet/core/razor-pages/razor-pages-convention-features) topic.
```
Comment (annmariehitchcock) [inline aspnetcore/razor-pages/razor-pages-conventions/samples/2.x/SampleApp/README.md:3]: Adding to broken link tab. Try https://learn.microsoft.com/aspnet/core/razor-pages/razor-pages-conventions

File: aspnetcore/razor-pages/razor-pages-conventions/samples/3.x/SampleApp/README.md [hunk 5]
```diff
@@ -1,12 +1,12 @@
 # ASP.NET Core Model Providers Sample
 
-This sample illustrates use of Razor Pages custom route and page model providers. This sample demonstrates the features described in the [Razor Pages route and app conventions](https://docs.microsoft.com/aspnet/core/razor-pages/razor-pages-convention-features) topic.
+This sample illustrates use of Razor Pages custom route and page model providers. This sample demonstrates the features described in the [Razor Pages route and app conventions](https://learn.microsoft.com/aspnet/core/razor-pages/razor-pages-convention-features) topic.
```
Comment (annmariehitchcock) [inline aspnetcore/razor-pages/razor-pages-conventions/samples/3.x/SampleApp/README.md:3]: Same as above, Adding to broken link tab. Try https://learn.microsoft.com/aspnet/core/razor-pages/razor-pages-conventions

**Comment by annmariehitchcock** [comment]
#docutune-ready

File: aspnetcore/blazor/components/cascading-values-and-parameters.md (hunk 1 above)
Comment (Rick-Anderson) [inline aspnetcore/blazor/components/cascading-values-and-parameters.md:745]: <a href="https://docs.microsoft.com/aspnet/" target="_blank">About</a>

**Review by Rick-Anderson — CHANGES_REQUESTED** [review]
revert

**Comment by alexbuckgit** [comment]
#docutune-hold

File: aspnetcore/blazor/components/cascading-values-and-parameters.md (hunk 1 above)
Comment (alexbuckgit) [reply aspnetcore/blazor/components/cascading-values-and-parameters.md]: Just reverting this one for the rebrand. Thanks for the feedback.

File: aspnetcore/fundamentals/middleware/extensibility/samples/3.x/MiddlewareExtensibilitySample/README.md (hunk 3 above)
Comment (alexbuckgit) [reply aspnetcore/fundamentals/middleware/extensibility/samples/3.x/MiddlewareExtensibilitySample/README.md:3]: Leaving for now

File: aspnetcore/razor-pages/razor-pages-conventions/samples/2.x/SampleApp/README.md (hunk 4 above)
Comment (alexbuckgit) [reply aspnetcore/razor-pages/razor-pages-conventions/samples/2.x/SampleApp/README.md:3]: Pre-existing issue, keeping for now.

File: aspnetcore/razor-pages/razor-pages-conventions/samples/3.x/SampleApp/README.md (hunk 5 above)
Comment (alexbuckgit) [reply aspnetcore/razor-pages/razor-pages-conventions/samples/3.x/SampleApp/README.md:3]: Pre-existing issue, keeping for now.

File: aspnetcore/blazor/components/cascading-values-and-parameters.md (hunk 1 above)
Comment (alexbuckgit) [reply aspnetcore/blazor/components/cascading-values-and-parameters.md]: Reverted

File: aspnetcore/fundamentals/middleware/extensibility/samples/2.x/MiddlewareExtensibilitySample/README.md (hunk 2 above)
Comment (alexbuckgit) [reply aspnetcore/fundamentals/middleware/extensibility/samples/2.x/MiddlewareExtensibilitySample/README.md:3]: Pre-existing issue, keeping for now.

**Comment by alexbuckgit** [comment]
@IEvangelist @Rick-Anderson 

Please merge - thanks!

#docutune-request-approval

**Review by guardrex — APPROVED** [review]

**Pull request merged**